### DISCUSSION TITLE
Welcome new committer: Liu Xun Zhuo

### DISCUSSION THREAD

**Discussion by seeflood** [mosn/layotto#541] [Announcements]
Hi Community,

On behalf of the Layotto PMC, I am pleased to announce that Liu Xun Zhuo a.k.a @Xunzhuo, has been voted in as Layotto committer.
![image](https://user-images.githubusercontent.com/26001097/166681103-a53a32a5-1a8d-4a5e-bab6-feb11fe255a1.png)

Liu Xun Zhuo has been making great contributions to Layotto. Please join me to say congratulations to him!

@Xunzhuo , would you please briefly introduce yourself to the community?

Best Regards.

**Comment by Xunzhuo** [comment]
Hi forks, It is my great honor and happy to onboard~

 I am working at Tencent Cloud Mesh Team, focusing on Service Mesh, API Gateway, especially in Extension and Telemetry Fields, exploring how to combine Observability with WASM and putting it into production, also putting efforts in eBPF to improve service mesh performance by reducing latency

Of course, I am a completely OSS lover, thanks for the recognition from the team, let us do something awesome in the future together:)

**Reply by khotyn** [reply]
Welcome, congrats!

**Reply by Xunzhuo** [reply]
Thanks @khotyn
